hexchat/README.md

```diff
@@ -0,0 +1,14 @@
+# HEXCHAT snap
+
+This project creates a working snap of hexchat.
+
+## Current state
+
+Working features: The snap can successfully install hexchat
+
+Known issues: We use a fork of hexchat (https://github.com/hikiko/hexchat) until
```

Comment:
I would word it like:

`Known issues: This snap currently uses a fork of hexchat (https://github.com/hikiko/hexchat) until`